workflow/python/ds-pack/v1/requirements.txt

```diff
@@ -5,3 +5,4 @@ snowflake==0.0.3
 snowflake-connector-python[pandas]==2.7.7
```

Comment:
Hey @saurabh0402 - i Guess we can remove this now as `snowflake-sqlalchemy` will install `snowflake-connector-python` as well.

Reply:
Think we need this to pin the version and also this will install the pandas-specific things as well 🤔